edgexfoundry/edgex-compose
# fix: Add env override for UI Service.Host

fixes #299

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
From compose builder using this branch
run `make run no-secty`
Verify access to UI from browser at `localhost:4000`
